From: Mathias Gibbens <gibmat@debian.org>
Description: Adjust import paths to reflect Debian packaging
Forwarded: not-needed
diff --git a/cmd/incusd/daemon.go b/cmd/incusd/daemon.go
index 139fe549b..59f347c02 100644
--- a/cmd/incusd/daemon.go
+++ b/cmd/incusd/daemon.go
@@ -23,7 +23,7 @@ import (
 	dqliteClient "github.com/cowsql/go-cowsql/client"
 	"github.com/cowsql/go-cowsql/driver"
 	"github.com/gorilla/mux"
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"golang.org/x/sys/unix"
 
 	internalIO "github.com/lxc/incus/v6/internal/io"
diff --git a/cmd/incusd/instance_console.go b/cmd/incusd/instance_console.go
index 1631c6146..d9685a4c8 100644
--- a/cmd/incusd/instance_console.go
+++ b/cmd/incusd/instance_console.go
@@ -17,7 +17,7 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/gorilla/websocket"
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"golang.org/x/sys/unix"
 
 	internalInstance "github.com/lxc/incus/v6/internal/instance"
diff --git a/cmd/incusd/main_forkconsole.go b/cmd/incusd/main_forkconsole.go
index 5befe7a35..a3091f513 100644
--- a/cmd/incusd/main_forkconsole.go
+++ b/cmd/incusd/main_forkconsole.go
@@ -7,7 +7,7 @@ import (
 	"strconv"
 	"strings"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"github.com/spf13/cobra"
 )
 
diff --git a/cmd/incusd/main_forkmigrate.go b/cmd/incusd/main_forkmigrate.go
index eb143e3b5..a37fe6d1a 100644
--- a/cmd/incusd/main_forkmigrate.go
+++ b/cmd/incusd/main_forkmigrate.go
@@ -6,7 +6,7 @@ import (
 	"os"
 	"strconv"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"github.com/spf13/cobra"
 )
 
diff --git a/cmd/incusd/main_forkstart.go b/cmd/incusd/main_forkstart.go
index 82568003a..3bff7d0c4 100644
--- a/cmd/incusd/main_forkstart.go
+++ b/cmd/incusd/main_forkstart.go
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"os"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"github.com/spf13/cobra"
 	"golang.org/x/sys/unix"
 
diff --git a/cmd/incusd/storage_volumes_snapshot.go b/cmd/incusd/storage_volumes_snapshot.go
index 0325164d9..fd049b843 100644
--- a/cmd/incusd/storage_volumes_snapshot.go
+++ b/cmd/incusd/storage_volumes_snapshot.go
@@ -13,7 +13,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/flosch/pongo2/v6"
+	"github.com/flosch/pongo2"
 	"github.com/gorilla/mux"
 
 	internalInstance "github.com/lxc/incus/v6/internal/instance"
diff --git a/cmd/lxc-to-incus/main_migrate.go b/cmd/lxc-to-incus/main_migrate.go
index 3ee5b02b5..759cac96a 100644
--- a/cmd/lxc-to-incus/main_migrate.go
+++ b/cmd/lxc-to-incus/main_migrate.go
@@ -10,7 +10,7 @@ import (
 	"strconv"
 	"strings"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"github.com/spf13/cobra"
 
 	incus "github.com/lxc/incus/v6/client"
diff --git a/cmd/lxc-to-incus/main_migrate_test.go b/cmd/lxc-to-incus/main_migrate_test.go
index b9d826dca..91cf8acb7 100644
--- a/cmd/lxc-to-incus/main_migrate_test.go
+++ b/cmd/lxc-to-incus/main_migrate_test.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"testing"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"github.com/stretchr/testify/require"
 )
 
diff --git a/cmd/lxc-to-incus/network.go b/cmd/lxc-to-incus/network.go
index 1d0752de5..320ea1852 100644
--- a/cmd/lxc-to-incus/network.go
+++ b/cmd/lxc-to-incus/network.go
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"strings"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 )
 
 func networkGet(container *liblxc.Container, index int, configKey string) map[string]string {
diff --git a/internal/server/device/proxy.go b/internal/server/device/proxy.go
index eb8293bf6..13661598e 100644
--- a/internal/server/device/proxy.go
+++ b/internal/server/device/proxy.go
@@ -13,7 +13,7 @@ import (
 	"strings"
 	"time"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 
 	"github.com/lxc/incus/v6/internal/linux"
 	"github.com/lxc/incus/v6/internal/server/apparmor"
diff --git a/internal/server/instance/drivers/driver_lxc.go b/internal/server/instance/drivers/driver_lxc.go
index b6d8cb9a0..4cfb3d007 100644
--- a/internal/server/instance/drivers/driver_lxc.go
+++ b/internal/server/instance/drivers/driver_lxc.go
@@ -27,11 +27,11 @@ import (
 	"time"
 
 	"github.com/checkpoint-restore/go-criu/v6/crit"
-	"github.com/flosch/pongo2/v6"
+	"github.com/flosch/pongo2"
 	"github.com/google/uuid"
 	"github.com/gorilla/websocket"
 	"github.com/kballard/go-shellquote"
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"github.com/pkg/sftp"
 	"golang.org/x/sync/errgroup"
 	"golang.org/x/sys/unix"
diff --git a/internal/server/instance/drivers/driver_qemu.go b/internal/server/instance/drivers/driver_qemu.go
index a8f180c1a..17880f4ee 100644
--- a/internal/server/instance/drivers/driver_qemu.go
+++ b/internal/server/instance/drivers/driver_qemu.go
@@ -31,7 +31,7 @@ import (
 	"time"
 	"unsafe"
 
-	"github.com/flosch/pongo2/v6"
+	"github.com/flosch/pongo2"
 	"github.com/google/uuid"
 	"github.com/gorilla/websocket"
 	"github.com/kballard/go-shellquote"
diff --git a/internal/server/instance/instance_interface.go b/internal/server/instance/instance_interface.go
index bde9ad2bb..41be4d5b5 100644
--- a/internal/server/instance/instance_interface.go
+++ b/internal/server/instance/instance_interface.go
@@ -8,7 +8,7 @@ import (
 	"os"
 	"time"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"github.com/pkg/sftp"
 	"google.golang.org/protobuf/proto"
 
diff --git a/internal/server/instance/instance_utils.go b/internal/server/instance/instance_utils.go
index e77f4a890..2c69df5f4 100644
--- a/internal/server/instance/instance_utils.go
+++ b/internal/server/instance/instance_utils.go
@@ -16,7 +16,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/flosch/pongo2/v6"
+	"github.com/flosch/pongo2"
 	"github.com/google/uuid"
 
 	incus "github.com/lxc/incus/v6/client"
diff --git a/internal/server/network/driver_ovn.go b/internal/server/network/driver_ovn.go
index adc37bea8..2b83ea231 100644
--- a/internal/server/network/driver_ovn.go
+++ b/internal/server/network/driver_ovn.go
@@ -18,7 +18,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/flosch/pongo2/v6"
+	"github.com/flosch/pongo2"
 	"github.com/mdlayher/netx/eui64"
 	ovsClient "github.com/ovn-org/libovsdb/client"
 	ovsdbModel "github.com/ovn-org/libovsdb/model"
diff --git a/internal/server/seccomp/seccomp.go b/internal/server/seccomp/seccomp.go
index b0bb47596..b760a1a98 100644
--- a/internal/server/seccomp/seccomp.go
+++ b/internal/server/seccomp/seccomp.go
@@ -465,7 +465,7 @@ import (
 	"time"
 	"unsafe"
 
-	liblxc "github.com/lxc/go-lxc"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
 	"golang.org/x/sys/unix"
 
 	"github.com/lxc/incus/v6/internal/linux"
diff --git a/internal/util/template.go b/internal/util/template.go
index 29eaa6675..132915654 100644
--- a/internal/util/template.go
+++ b/internal/util/template.go
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/flosch/pongo2/v6"
+	"github.com/flosch/pongo2"
 )
 
 // RenderTemplate renders a pongo2 template with nesting support.
