From 27232680a75f5c96bae1f39dc085b7e59b882815 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Graber?= <stgraber@stgraber.org>
Date: Mon, 10 Nov 2025 05:42:13 -0500
Subject: [PATCH] incusd/patches: Fix incorrect error check in permission patch
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Stéphane Graber <stgraber@stgraber.org>
---
 cmd/incusd/patches.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmd/incusd/patches.go b/cmd/incusd/patches.go
index e6bef88e7ab..c64c9bbf828 100644
--- a/cmd/incusd/patches.go
+++ b/cmd/incusd/patches.go
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io/fs"
 	"net/http"
 	"os"
 	"path/filepath"
@@ -1638,7 +1639,7 @@ func patchDefaultStoragePermissions(_ string, d *Daemon) error {
 				path := filepath.Join(storagePools.GetStoragePoolMountPoint(pool), volDir)
 
 				err := os.Chmod(path, volEntry.Mode)
-				if err != nil && !os.IsExist(err) {
+				if err != nil && !errors.Is(err, fs.ErrNotExist) {
 					return fmt.Errorf("Failed to set directory mode %q: %w", path, err)
 				}
 			}
